documentation/TechnicalDesignDocument.tex

```diff
@@ -181,34 +199,62 @@
-However, there is no functionality in libGDX for finding the face of a \code{SelectableInstance}.
-To remedy this a new addition was made to the \code{SelectableInstance}: Joints.
-Every \code{SelectableInstance} has a list of Joints which contain a \code{Vector3} which states the location of the joints of the \code{SelectableInstance} that can be interacted with.
-When the ray wants to interact with a \code{SelectableInstance}(See \href{\clickup{2e29e}}{\#2e29e}) it returns the location of the intersect point.
+    Using the \code{BoundingBox} you can find which \code{SelectableInstance} you are close to fairly easily.
+    However, there is no functionality in libGDX for finding the face of a \code{SelectableInstance}.
+    To remedy this a new addition was made to the \code{SelectableInstance}: Joints.
+    Every \code{SelectableInstance} has a list of Joints which contain a \code{Vector3} which states the location of the joints of the \code{SelectableInstance} that can be interacted with.
+    When the ray wants to interact with a \code{SelectableInstance}(See \href{\clickup{2e29e}}{\#2e29e}) it returns the location of the intersect point.
     The points get translated and compared to one another to find the closest face to the intersect point.
-Then the new \code{SelectableInstance} can be added.
+    Then the new \code{SelectableInstance} can be added.
     The functionality is not wholly completed.
-It uses the face of the target \code{SelectableInstance} and that of the origin \code{SelectableInstance} to position the new \code{SelectableInstance} in the correct spot.
+    It uses the face of the target \code{SelectableInstance} and that of the origin \code{SelectableInstance} to position the new \code{SelectableInstance} in the correct spot.
     In the future it should also rotate so that the joints properly face one another.
 
 
 
     \newpage
     %------------------------------------------------------------------------------------------------------------------------------------------------------- Build Automation
-    \section{Continous Intergration}
+    \section{Continous Integration}
     In this section build automation and its constituent components will be discussed.
     How we applied it, the software used and the targets and platforms supported are among a few.
 
+    The requirements for the project contain several important matters.
+    The project needs to be able to be developed and built from several OSes.
+    It needs to be able to switch hands between multiple developers easily, due to it being open source.
+    Obviously it is also important that those developers can also see that the build they are working from is being tested properly.
+    Because of this the team, together with Chide, decided that a continuous integration solution was necessary, and have been working on setting this up during the first part of the project.
```

Comment:
Chide demanded this, it was one of his first requirements. We had little say.

Reply:
You sure? Then my memory is doing a dumb. Feel free to rewrite.

Reply:
It is your PR

Reply:
done